jumpcloud-ADMU/Powershell/Tests/manualTests/selectionForm.feature

```diff
@@ -0,0 +1,217 @@
+        And the "Migrate Profile" button is pressed
+        Then a windows form window should appear and warn: The JumpCloud agent is not installed and to also enter your connectKey
+
+    Scenario: Disallows migration for usernames that are not found in the JumpCloud organization
+        Given the JumpCloud agent is installed on the device
+        And The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "AutoBind JCUser" checkbox is selected
+        And a valid API Key is pasted in the APIKey text field
+        And a JumpCloud username that does not exist in the JumpCloud Organization is specified in the username text box
+        And a valid password is specified in the password text box
+        And the "Migrate Profile" button is pressed
+        Then a windows form window should appear and note the specified username was not found in the JumpCloud Organization
+
+    Scenario: Prompts warning for migration for usernames that have a localSystemUsername specified in the JumpCloud Organization
+        Given the JumpCloud agent is installed on the device
+        And The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "AutoBind JCUser" checkbox is selected
+        And a valid API Key is pasted in the APIKey text field
+        And a JumpCloud username that does exist in the JumpCloud Organization is specified in the username text box
+        And a valid password is specified in the password text box
+        And the "Migrate Profile" button is pressed
+        Then a windows form window should appear and note that the specified user has a local account username/ can select OK or Cancel to return to selection form
+
+    Scenario: Prompts warning for migration for usernames that have a localSystemUsername specified in the JumpCloud Organization and match a local user on the system
+        Given a local user is created on the system matching some user's localUsername field in JumpCloud
+        And the JumpCloud agent is installed on the device
+        And The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "AutoBind JCUser" checkbox is selected
+        And a valid API Key is pasted in the APIKey text field
+        And a JumpCloud username who has a localUsername field specified (that matches the local user created for this test) that does exist in the JumpCloud Organization is specified in the username text box
+        And a valid password is specified in the password text box
+        And the "Migrate Profile" button is pressed
+        Then a windows form window should appear and note migration can not continue when the specified user has a localUsername that matches a user that exists on the device
```

Comment:
Same as above tests: 
<img width="1189" alt="image" src="https://github.com/user-attachments/assets/84d233b7-506e-4de0-acdb-f9713a17a5f5" />

Reply:
disregard, form shows the correct message